src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -69,11 +70,8 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
     this.options = ComponentOptions.initComponentOptions(element, QuerySuggestPreview, options);
 
     if (!this.options.resultTemplate) {
-      // TODO: Add a default template
-      this.logger.warn(
-        `Specifying a result template is required for the 'QuerySuggestPreview' component to work properly. See `,
-        `https://docs.coveo.com/340/#providing-query-suggestion-result-previews`
-      );
+      this.logger.warn(`No template was provided for ${QuerySuggestPreview.ID}, a default template was used instead.`);
```

Comment:
It seems there is no documentation for the `resultTemplate` option ? 

There should be an explanation that comes with the option that explains how to use it, some examples on how to perform "standard" customization (eg: changing the size of the preview image), and that a fallback template is used internally if none is given.

@fbeaudoincoveo any opinion on how this should be presented ?

Reply:
I added a short bit of documentation in https://github.com/coveo/search-ui/pull/1272, but it doesn't say much and it doesn't tell anything about the default value.

I'll add documentation here when @fbeaudoincoveo responds.